docs/General-Workflow.md

```diff
@@ -1,26 +1,102 @@
 This describes an example of how to iteratively edit the splat segments config, when decompiling
 
+(If you have no idea what this is about, please head over to the [Quickstart](https://github.com/ethteck/splat/wiki/Quickstart) to get an initial configuration for your ROM.)
+
 # 1 Initially
 
-Assuming that after you split segments you start from a code segment which subsegments include
+Assuming that after succesfully following the [Quickstart](https://github.com/ethteck/splat/wiki/Quickstart), you get an initial configuration like the one below:
+```yaml
+- name: main
+  type: code
+  start: 0x1060
+  vram: 0x80070C60
+  follows_vram: entry
+  bss_size: 0x3AE70
+  subsegments:
+      - [0x1060, asm]
+      # ... a lot of additional `asm` sections
+      # This section is found out to contain __osViSwapContext
+      - [0x25C20, asm, "energy_orb_wave"]
+      # ... a lot of addtiional `asm` sections
+      - [0x2E450, data]
+
+      - [0x3E330, rodata]
+      # ... a lot of addtional `rodata` sections
+      - { start: 0x3F1B0, type: bss, vram: 0x800E9C20 }
+
+- [0x3F1B0, bin]
+```
+
+## 1.1 Match rodata to asm sections
+
+In order to simplify decompilation, it's good practice to start pairing `rodata` sections with `asm` sections.
+
+`splat` gives hints about what `rodata` is used in which `asm` segment.
+
+The hint look like:
+```
+Rodata segment '3EE10' may belong to the text segment 'energy_orb_wave'
+    Based on the usage from the function func_0xXXXXXXXX to the symbol D_800AEA10
+```
+
+To pair these two sections, simply add the name of the suggested text (`asm`) segment to the `rodata` segment:
+
+```yaml
+- [0x3EE10, rodata, "energy_orb_wave"]
+```
+
+### Useful knowledge about splitting
+
+#### Multiple `rodata`
+Using the following configuration:
 ```yaml
-- [0x42100, bin]
+# ...
+- [0x3E900, rodata]
+- [0x3E930, rodata]
+# ...
 ```
 
+`splat` outputs a hint that doesn't immediately seem to make sense:
+
+```
+Rodata segment '3E900' may belong to the text segment '16100'
+    Based on the usage from the function func_80085DA0 to the symbol jtbl_800AE500
+
+Rodata segment '3E930' may belong to the text segment '16100'
+    Based on the usage from the function func_800862C0 to the symbol jtbl_800AE530
+```
+
+This hint tells you that `splat` thinks one text (`asm`) segment seems to have two `rodata` sections. This usually means that there should not be a split at `0x3E930`, since one text segment should only have one `rodata` segment. 
```

Comment:
This statement is half true, this could also mean there's a file split in the `asm` section.

Reply:
Perfect, I added it as a commit!